README.md

```diff
@@ -1,13 +1,86 @@
-======
-cf-plot
-======
+# cf-plot
 
-cf-plot is a set of Python plotting routines for the contour, vector and line plots that climate
-researchers commonly make.  
+## Code-light plotting for earth science and aligned research
 
+### Overview
 
-Documentation
-=============
-Please refer to the cf-plot homepage http://ajheaps.github.io/cf-plot
+**cf-plot allows you to produce and customise publication-quality contour,
+vector, line and more plots with the power of Python,
+[matplotlib](https://matplotlib.org/),
+[Cartopy](https://scitools.org.uk/cartopy/docs/latest/) and
+[cf-python](https://ncas-cms.github.io/cf-python/), in as few lines of code
+as possible.**
 
+It is designed to be a useful visualisation tool for environmental, earth
+and aligned science, for example to facilitate climate and meteorological
+research. cf-plot is developed and maintained by the
+[NCAS-CMS](https://cms.ncas.ac.uk/index.html) group, part of
+[NCAS](https://ncas.ac.uk/).
 
+
+### Brief Demonstration
+
+In as little as four lines of Python including imports, using `cf-plot` you can
+for example produce a contour plot showing a 2D subspace of a netCDF dataset:
+
+```python
+import cf
+import cfplot as cfp
+f = cf.read('<dataset name>.nc')[0]  # picks out a read-in field of the dataset
+cfp.con(f.subspace(time=<chosen time value>))  # creates a contour plot of the field at that time value
+```
+
+
+### Examples Gallery
+
+A gallery of outputs made with cf-plot, showcasing a range of plotting
+possibilities with links to relevant documentation pages and to example code,
+can be found
+[on this dedicated page within the documentation](http://ajheaps.github.io/cf-plot/gallery.html),
+as illustrated in this (static) image:
```

Comment:
```suggestion
The [gallery of outputs made with cf-plot](http://ajheaps.github.io/cf-plot/gallery.html) showcases a range of plotting
possibilities with links to relevant documentation pages and to example code,
as illustrated in this (static) image:
```

Reply:
... thinking more, maybe you could move the image to the top of the README, so that it gets seen by the don't-scroll-down crowd. Just a thought.

Reply:
I did wonder whether this would be best, but also wanted to highlight to *that* crowd how few lines are required to get a plot out and put that first. However, now you mention it, they are more likely to scroll further past an image, so I agree with your suggestion. Thanks, I'll promote it to the first section, then mark this comment as resolved.

Reply:
Actually, re-reading I see you were probably instead proposing to move the image only, not that whole section. Might be best, good thinking. I'll do that and point to it from the 'Examples Gallery' section, left in the same place.

Reply:
See [3cb767e](https://github.com/NCAS-CMS/cf-plot/pull/30/commits/3cb767e65d6d69c86312b03d814091a1a5870e51) where I've moved the image (only) to the top.